### PR TITLE
S4jaky1

### PR DESCRIPTION
**Pull request by krandou**
# WELCOME TO DEVOPS-EASY-LEARNING S3 PIPELINE  PULL REQUEST SYSTEM 
# Description of the change

> Detailed description here

## Checks and balances

- [ ] Tests added and run
- [ ] (External) Interfaces documented
- [ ] Has security implications (describe below)

## Type of change

- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing
functionality to not work as expected)

## Related stories, issues and pulls

> Links to Pivotal Tracker stories, JIRA tickets or other pull requests here

## Security considerations

> Describe any security impact this change may have

FROM HERE, LIFE AND  CAREER CHANGE ARE  POSSIBLE

### CODE REVIEW

**Comment by s4lydiane** [comment]
good job

**Review by s4lydiane — APPROVED** [review]

File: restaurant-app/yelb-appserver/README.md [hunk 1]
```diff
@@ -1,3 +1,3 @@
 This is the application server. It's a Ruby/Sinatra application that exposes a number of APIs that are consumed by the UI (or via curl if you fancy that). 
 
-Originally this application was included in a single file (`yelb-appserver.rb`). This has been since refactored by extracting the single API definition in their separate adapters and modules. This made the transition to Lambda/Serverless easier (one lambda per API definition). This hasn't changed the other deployment models (containers and instances) because those models still launch the `yelb-appserver.rb` main application which imports the modules instead of having everything in a single file (as it was conceived originally).
\ No newline at end of file
+Originally this application was included \er (one lambda per API definition). This hasn't changed the other deployment models (containers and instances) because those models still launch the `yelb-appserver.rb` main application which imports the modules instead of having everything in a single file (as it was conceived originally). vn  nvn m fhbmn
```
Comment (Agnonyao) [inline restaurant-app/yelb-appserver/README.md:3]: nice

**Review by Agnonyao — APPROVED** [review]

File: restaurant-app/yelb-appserver/README.md [hunk 2]
```diff
@@ -1,3 +1,3 @@
 This is the application server. It's a Ruby/Sinatra application that exposes a number of APIs that are consumed by the UI (or via curl if you fancy that). 
 
-Originally this application was included in a single file (`yelb-appserver.rb`). This has been since refactored by extracting the single API definition in their separate adapters and modules. This made the transition to Lambda/Serverless easier (one lambda per API definition). This hasn't changed the other deployment models (containers and instances) because those models still launch the `yelb-appserver.rb` main application which imports the modules instead of having everything in a single file (as it was conceived originally).
```
Comment (Agnonyao) [inline restaurant-app/yelb-appserver/README.md:3]: it's included

**Review by Agnonyao — APPROVED** [review]

**Comment by s4zongo** [comment]
hum

**Pull request closed without merging**